# Example configuration for one camera
# Expected to be used with camera.launch.py using default name and namespace

/ifm3d/camera:
  ros__parameters:
    config_file: "/this/path/will/be/overridden/from/the/lauchfile/because/yaml/files/cant/resolve/find-pkg-share/command"
    buffer_id_list:
      - CONFIDENCE_IMAGE
      - EXTRINSIC_CALIB
      - INTRINSIC_CALIB
      - INVERSE_INTRINSIC_CALIBRATION
      - JPEG_IMAGE
      - NORM_AMPLITUDE_IMAGE
      - RADIAL_DISTANCE_IMAGE
      - RGB_INFO
      - TOF_INFO
      - XYZ
    ip: 192.168.0.69
    pcic_port: 50010
    tf:
      base_frame_name: "ifm_base_link"
      mounting_frame_name: "camera_mounting_link"
      optical_frame_name: "camera_optical_link"
      publish_base_to_mounting: true
      publish_mounting_to_optical: true
    use_sim_time: false
    xmlrpc_port: 80
